syntax = "proto3";

package health;

message HealthRequest {
    string contents = 1;
}

message HealthResponse {
    string message = 1;
}

service HealthService {
    rpc CheckHealth(HealthRequest) returns (HealthResponse);
}
